test/extended/templates/templateinstance_cross_namespace.go

```diff
@@ -28,6 +28,9 @@ var _ = g.Describe("[templates] templateinstance cross-namespace test", func() {
 	)
 
 	g.It("should create and delete objects across namespaces", func() {
+		//TODO - when https://github.com/kubernetes-incubator/service-catalog/pull/939 sufficiently progresses, we should see about removing
+		// this add-role-to-user, as it may not be required, though cross namespace access may differentiate from the other applications of
+		// add-role-to-user in the template extended tests
```

Comment:
this should still be needed even then...  there's no reason cli1 user should be able to access resources in the cli2 namespace without it.  Am I missing something about why we'd expect this behavior to change?